website/docs/plugin/how-terraform-works.mdx

```diff
@@ -110,42 +110,3 @@ This behavior only applies to providers whose _only_ acceptable versions are in
-
-Major versions of the protocol delineate Terraform CLI and Terraform Plugin compatibility. Minor versions of the protocol are additive. The protocol is implemented in [Protocol Buffers](https://developers.google.com/protocol-buffers) and [gRPC](https://grpc.io/), with the canonical source for protocol definitions located in the [Terraform CLI repository](https://github.com/hashicorp/terraform/tree/main/docs/plugin-protocol).
-
-### Protocol Version 6
-
-Protocol version 6 is compatible with Terraform CLI version 1.0 and later. Protocol version 6 includes all version 5 functionality for providers, plus:
-
-* **Nested Attributes**: Define `SchemaAttribute` with the `NestedType` field.
-    * Enable practitioners to use easier [argument syntax](/language/syntax/configuration#arguments) instead of [block syntax](/language/syntax/configuration#blocks).
-    * Configure value sensitivity on individual nested attributes, rather than an entire read-only (`Computed` only) attribute.
-
-Implementations include:
-
-* [terraform-plugin-framework](/plugin/framework): A higher-level SDK that makes Terraform provider development easier by abstracting implementation details.
-* [terraform-plugin-go tf6server](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-go/tfprotov6/tf6server): A lower-level SDK to develop Terraform providers for more advanced use cases.
-* [tf5to6server](/plugin/mux/translating-protocol-version-5-to-6): A package to translate protocol version 5 providers into protocol version 6.
-* [tf6muxserver](/plugin/mux/combining-protocol-version-6-providers): A package to combine multiple protocol version 6 providers.
-
-### Protocol Version 5
-
-Protocol version 5 is compatible with Terraform CLI version 0.12 and later.
-
-Implementations include:
-
-* [terraform-plugin-framework](/plugin/framework): A higher-level SDK that makes Terraform provider development easier by abstracting implementation details.
-* [terraform-plugin-sdk/v2](/plugin/sdkv2): A higher-level SDK that makes Terraform provider development easier by abstracting implementation details.
-* [terraform-plugin-go tf5server](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server): A lower-level SDK to develop Terraform providers for more advanced use cases.
-* [tf6to5server](/plugin/mux/translating-protocol-version-6-to-5): A package to translate protocol version 6 providers into protocol version 5.
-* [tf5muxserver](/plugin/mux/combining-protocol-version-5-providers): A package to combine multiple protocol version 5 providers.
-
-[0]: https://en.wikipedia.org/wiki/Static_build#Static_building
-
-[1]: https://golang.org/
```

Comment:
I think these need to stay on this page (or be changed to inline links instead of detached links)